src/components/theme.js

```diff
@@ -78,7 +78,7 @@ let theme = createMuiTheme({
           fontSize: "0.7rem",
         },
       },
-      p: {
+      body1: {
```

Comment:
Is there a specific reason to have`body1` instead of `p`? Why `body1`?

Reply:
P gives a key error, body1 is the appropriate key on material Ui

Reply:
Gotcha